draft-ietf-tls-esni.md

```diff
@@ -1557,18 +1580,33 @@ for ExtensionType (defined in {{!RFC8446}}):
 
 1. encrypted_client_hello(0xfe0a), with "TLS 1.3" column values set to
    "CH, EE", and "Recommended" column set to "Yes".
-2. ech_is_inner (0xda09), with "TLS 1.3" column values set to
+1. ech_is_inner (0xda09), with "TLS 1.3" column values set to
    "CH", and "Recommended" column set to "Yes".
-3. ech_outer_extensions(0xfd00), with the "TLS 1.3" column values set to "",
+1. ech_outer_extensions(0xfd00), with the "TLS 1.3" column values set to "",
    and "Recommended" column set to "Yes".
 
 ## Update of the TLS Alert Registry {#alerts}
 
-IANA is requested to create an entry, ech_required(121) in the existing registry
-for Alerts (defined in {{!RFC8446}}), with the "DTLS-OK" column set to
+IANA is requested to create an entry, ech_required(121) in the existing
+registry for Alerts (defined in {{!RFC8446}}), with the "DTLS-OK" column set to
 "Y".
 
-# ECHConfig Extension Guidance {#config-extensions-guidance}
+## ECHConfig Extension Registry
+
+This document defines a new ECHConfig extension registry that is maintained
+by IANA. Values with the first byte in the range 0-253 (decimal) are assigned
+via Specification Required {{!RFC8126}}.  Values with the first byte 254 or 255
+(decimal) are reserved for Private Use {{!RFC8126}}.
+
+This registry has a "Recommended" and "Mandatory" column. The registry is
+initially populated with a single entry for the "public_name" extension with
+"Recommended" and "Mandatory" values set to "Y" and "Y", respetively.
```

Comment:
(Just starting a discussion. I don't feel strongly either way, so whatever you think is best.)

It's interesting that we have a "Mandatory" column when it's purely a function of the value. On the one hand, it avoids readers needing to look at the constant. On the other hand, it might give the impression you can arbitrarily assign that column and you can't.

Maybe if the registry had a note at the top explaining how the mandatory bit works?

WDYT?

Reply:
That's a good point -- I tried to add some text which clarifies that the value has implications on users.